index.js

```diff
@@ -40,6 +42,24 @@ function increaseSpecifityOfRule(rule, opts) {
 	}
 }
 
+function isDisabled(node, status) {
```

Comment:
New signature, `checkForEnableDisableComments(fromNode, toNode)`

Reply:
I don't think a signature like that makes sense in this sense. If we are already holding the previous result in `isPluginDisabled`, then we only have to check above the current node.

I've updated the function name and made it to continue traversing up comments until it finds one that matches OR no more comments are found.